js/flightlog.js

```diff
@@ -1180,6 +1181,12 @@ FlightLog.prototype.rcCommandRawToThrottle = function(value) {
     return Math.min(Math.max(((value - this.getSysConfig().minthrottle) / (this.getSysConfig().maxthrottle - this.getSysConfig().minthrottle)) * 100.0, 0.0),100.0);
 };
 
+// rcCommandThrottle back transform function
+FlightLog.prototype.ThrottleTorcCommandRaw = function(value) {
+    // Throttle displayed as percentage
+    return  value/100*(this.getSysConfig().maxthrottle - this.getSysConfig().minthrottle) + this.getSysConfig().minthrottle;
```

Comment:
:space_invader: give operators space for better readability
```suggestion
    return  value / 100 * (this.getSysConfig().maxthrottle - this.getSysConfig().minthrottle) + this.getSysConfig().minthrottle;
```